iothub_client/src/iothub_client_core.c

```diff
@@ -104,7 +104,7 @@ typedef struct HTTPWORKER_THREAD_INFO_TAG
     CALLBACK_TYPE_INPUTMESSAGE
 
 MU_DEFINE_ENUM(USER_CALLBACK_TYPE, USER_CALLBACK_TYPE_VALUES)
```

Comment:
>USER_CALLBACK_TYPE [](start = 15, length = 18)

Probably need to change this here and in the Iothub_client_core to use WITHOUT_INVALID